firmware/ec/Makefile

```diff
@@ -42,43 +51,56 @@ CONFIGURO = $(XDCTOOLS_DIR)/xs --xdcpath="$(XDCPATH)" \
 		xdc.tools.configuro $(CONFIGURO_OPTS)
 
 # Find all C source/object files.
-SRC_FILE = $(shell find . -name '*.c')
-MAIN_OBJS = $(patsubst %.c,%.o,$(wildcard *.c */*.c))
+SRC_FILE = $(shell find . -name '*.c' ! -path './test/*' ! -path './$(OUT)*')
+MAIN_OBJS = $(SRC_FILE:.c=.o)
 
 CC = $(TOOLCHAIN)/bin/arm-none-eabi-gcc
-CFLAGS = -Wall -mcpu=cortex-m4 -mthumb -mabi=aapcs -mapcs-frame @$(CONFIG)/compiler.opt -O3 -g
-CFLAGS += -DPART_$(MCU) -c -I$(TIVAWARE_PATH) -I$(OCWARE_DIR)/src
-CFLAGS += -I$(XDC_PATH) -I$(BIOS_PATH) -I$(TIDRIVER_PATH) -I./
-CFLAGS += -I$(OCWARE_DIR)/src/inc
-CFLAGS += -I./ -I$(NDK_PATH)
-CFLAGS += -I$(TIVAWARE_DIR)
+CFLAGS = -Wall -mcpu=cortex-m4 -mthumb -mabi=aapcs -mapcs-frame @$(OUT)/$(CONFIG)/compiler.opt -O3
```

Comment:
Maybe this is not the right forum to ask this question, but I'm curious to know why all optimizations have been turned on with "-O3".  No need to change this setting here, but something to consider as a discussion point.

Reply:
Not sure, I just left it in there because that was the previous setting.